Remove duplicate icons

newhoa opened the pull request:
Remove icons that were duplicated in different folders. Fix missing symlinks to `openoffice4-base` in the process.

ochosi commented:
@newhoa  This PR currently has a conflict that you will have to solve locally and then force-push to update

newhoa commented:
> @newhoa This PR currently has a conflict that you will have to solve locally and then force-push to update

@ochosi, Ok, fixed conflict and rebased. Should be good. Thanks!

ochosi commented on elementary-xfce/actions/symbolic/crosshairs.svg:
Is this one not used for anything? Thought it might be used as a mouse cursor or for the headerbar button in taskmanager.

newhoa replied:
taskmanager uses the non-symbolic icon, so the symbolic one shouldn't be used anywhere. Sorry, this probably shouldn't be in this PR since it's not technically a duplicate but if you're okay with deleting it here it should be fine.

ochosi replied:
Ok

ochosi reviewed (approved):
LGTM
